tests/python/frontend/mxnet/test_graph.py

```diff
@@ -117,3 +166,5 @@ def relay_compose(F, **kwargs):
     test_dcgan()
     test_squeezenet()
     test_inception_v3()
+    test_foreach_map()
+    test_foreach_fold()
```

Comment:
Could you also add foreach test with both outputs and tests from example 3 in https://mxnet.incubator.apache.org/versions/master/tutorials/control_flow/ControlFlowTutorial.html ?